plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/DeltaLakeSchemaSupport.java

```diff
@@ -88,19 +102,25 @@ public static boolean isAppendOnly(MetadataEntry metadataEntry)
         return Boolean.parseBoolean(metadataEntry.getConfiguration().getOrDefault(APPEND_ONLY_CONFIGURATION_KEY, "false"));
     }
 
+    public static ColumnMappingMode getColumnMappingMode(MetadataEntry metadata)
```

Comment:
This method seems to be used only in `io.trino.plugin.deltalake.transactionlog.DeltaLakeSchemaSupport#extractSchema`.

Why do we want to have `UNKNOWN` as a field in `ColumnMappingMode` if it is only used privately?

Reply:
The purpose of `UNKNOWN` is not throwing an exception when Delta Lake adds new mode in the future. 
By the way, I expect that #12621 goes first.